CRM/Upgrade/Incremental/php/FiveTwentyEight.php

```diff
@@ -25,10 +25,16 @@ class CRM_Upgrade_Incremental_php_FiveTwentyEight extends CRM_Upgrade_Incrementa
    * @param null $currentVer
    */
   public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL) {
-    // Example: Generate a pre-upgrade message.
-    // if ($rev == '5.12.34') {
-    //   $preUpgradeMessage .= '<p>' . ts('A new permission, "%1", has been added. This permission is now used to control access to the Manage Tags screen.', array(1 => ts('manage tags'))) . '</p>';
-    // }
+// Example: Generate a pre-upgrade message.
+    if (!function_exists('civi_wp')) {
+      //exit
+    }
+    elseif ($rev == '5.28.alpha1') {
+      $preUpgradeMessage .= '<p>' . ts('Starting with version 5.29.0, CiviCRM on WP will
+        automate the determination of the civicrm files directory. If you have a non-standard
```

Comment:
Capitalize CiviCRM Files?

Reply:
I was conflicted between that and referencing the `[civicrn.files]` format.   What do you think most users would recognize and understand?